pkg/issuercerts/issuercerts.go

```diff
@@ -180,7 +180,7 @@ func (ic *IssuerCertData) VerifyAndBuildCreds(dnsName string) (*tls.Cred, error)
 		return nil, err
 	}
 
-	if err := creds.Verify(anchors, dnsName, time.Time{}); err != nil {
+	if err := creds.Verify(anchors, "", time.Time{}); err != nil {
```

Comment:
Thought we could get rid of the `dnsName` argument totally here but the same func is used at other places to verify the proxy-injector's, etc leaf certificates where we do need to verify the dns name.